web_src/css/shared/settings.css

```diff
@@ -0,0 +1,20 @@
+details.toggleable-item summary {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+/* for specificity to outweigh the rule in base.css */
+.ui.menu details.toggleable-item {
+    user-select: none;
+}
+
+details.toggleable-item summary::after {
+    transition: 0.2s;
+    content: '\203A'; /* chevron right */
+    font-size: 20px;
+}
+
+details.toggleable-item[open] summary::after {
+    transform: rotate(90deg);
```

Comment:
I think you want to add `transform-origin: center;` so that the rotation origin is centered on the icon.